src/Compilers/CSharp/Test/Semantic/Semantics/AmbiguousOverrideTests.cs

```diff
@@ -139,10 +139,7 @@ public class Derived2 : Derived<int>
             comp.VerifyDiagnostics(
                 // (4,26): error CS0462: The inherited members 'Derived<TInt>.Method(long, TInt)' and 'Derived<TInt>.Method(long, int)' have the same signature in type 'Derived2', so they cannot be overridden
                 //     public override void Method(long l, int i) { }  //CS0462 and CS1957
-                Diagnostic(ErrorCode.ERR_AmbigOverride, "Method").WithArguments("Derived<TInt>.Method(long, TInt)", "Derived<TInt>.Method(long, int)", "Derived2").WithLocation(4, 26),
-                // (4,26): warning CS1957: Member 'Derived2.Method(long, int)' overrides 'Derived<int>.Method(long, int)'. There are multiple override candidates at run-time. It is implementation dependent which method will be called.
```

Comment:
>// (4,26): warning CS1957: Member 'Derived2.Method(long, int)' overrides 'Derived<int>.Method(long, int)'. There are multiple override candidates at run-time. It is implementation dependent which method will be called. [](start = 16, length = 218)

In order to confirm that there is indeed no problem now, we should test runtime behavior. Including scenario when the order of virtual methods declared in the ```Base``` is different.

Reply:
That is tested in `TestAmbiguousOverridesRefOut` in `CodeGenOverridingAndHiding`.

---
In reply to: [442518510](https://github.com/dotnet/roslyn/pull/44025#discussion_r442518510) [](ancestors = 442518510)